dist/redhat/scylla.spec

```diff
@@ -152,6 +152,7 @@ rm -rf $RPM_BUILD_ROOT
 %ghost %config /etc/systemd/system/var-lib-systemd-coredump.mount
 %ghost /etc/systemd/system/scylla-cpupower.service
 %ghost %config /etc/systemd/system/var-lib-scylla.mount
+%ghost /etc/sysctl.d/99-scylla-perfevent.conf
```

Comment:
Should be in kernel-conf subpackage.

Reply:
Moved `%ghost /etc/sysctl.d/99-scylla-perfevent.conf` to scylla-kernel-conf.
Also moved 99-scylla-perfevent.conf creation from `%post server` to `%post kernel-conf`, otherwise it will be inconsistent.